src/pages/news/news-detail/news-detail.js

```diff
@@ -115,4 +118,14 @@ const NewsDetailPage = ({ match }) => {
   );
 };
 
+const newsDetailPropTypes = {
+  params: PropTypes.shape({
```

Comment:
The actual prop is `match` not the `params`. `params` exists inside the `match` object

Reply:
done